Makefile

```diff
@@ -22,3 +22,7 @@ container: build
 
 container-run: container
 	docker run --name scanner-trivy --rm -d -p 8080:8080 $(IMAGE)
+
+lint:
+	curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOPATH)/bin v1.21.0
```

Comment:
For those who do not set `GOPATH` env `make lint` fails. I was the first one actually who bumped into that ;)
That said I'd rather install linter's binary to the `./bin` directory without relying on the environment config.

Also let's move the installation to a separate Make goal, e.g. setup. You install once, but run the actual linter multiple times:

```Makefile
setup:
       curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s v1.21.0
.PHONY: setup
```

Also make sure that you run `make setup` on Travis CI, e.g.

```yaml
install:
  - make setup
```